src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -1194,6 +1196,13 @@ static void maybe_start_some_streams(grpc_chttp2_transport* t) {
         GPR_INFO, "HTTP:%s: Allocating new grpc_chttp2_stream %p to id %d",
         t->is_client ? "CLI" : "SVR", s, t->next_stream_id));
 
+    if (t->goaway_error != GRPC_ERROR_NONE) {
+      gpr_log(GPR_ERROR,
+              "CAUTION: Starting new streams after receiving a GOAWAY frame. "
```

Comment:
This seems overly verbose to always report, especially with GRP_ERROR, since it's really a completely normal condition.  Maybe just log this under a tracer?

Reply:
Given that the HTTP2 spec says, "Receivers of a GOAWAY frame MUST NOT open
   additional streams on the connection, although a new connection can
   be established for new streams.", this is NOT a normal condition, and would be considered as bad behavior by a client.